src/main/java/sigma/internship/petProject/controller/MoneyBalanceController.java

```diff
@@ -0,0 +1,32 @@
+package sigma.internship.petProject.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Do not use wildcard imports in a project.
You can set up Intellij IDE to not use wildcard imports: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcards-for-class

Reply:
This issue is resolved for the controllers